### PR TITLE
Move from kc/konnect to libregraph/lico

### PR DESCRIPTION
**Pull request by rhafer**
This switches the oidc implementation to libregraph/lico, which for now
is just a rename of kc/konnect. As there is no working lico release yet
(lico 0.34.0 wasn't fully renamed), we point it to the current master.

### CODE REVIEW

**Review by butonic — APPROVED** [review]

**Comment by butonic** [comment]
could you search for `konnect` and replace occurrences with meaningful wording where possible, eg. the docs / markdown files, but I also see Dockerfiles and Makefile hits...

**Comment by rhafer** [comment]
> could you search for `konnect` and replace occurrences with meaningful wording where possible, eg. the docs / markdown files, but I also see Dockerfiles and Makefile hits...

Done

File: idp/docker/Dockerfile.linux.arm64 [hunk 1]
```diff
@@ -7,7 +7,7 @@ RUN apk update && \
 	echo 'hosts: files dns' >| /etc/nsswitch.conf
 
 LABEL maintainer="ownCloud GmbH <devops@owncloud.com>" \
-  org.label-schema.name="oCIS Konnectd" \
```
Comment (rhafer) [inline idp/docker/Dockerfile.linux.arm64:10]: @butonic Any idea if these labels are referenced anywhere? Is it safe to just rename them?

Comment (wkloucek) [reply idp/docker/Dockerfile.linux.arm64:10]: yeah can just be renamed :-)

**Pull request merged**